db/db_impl/db_impl_open.cc

```diff
@@ -1737,8 +1737,9 @@ Status DBImpl::WriteLevel0TableForRecovery(int job_id, ColumnFamilyData* cfd,
           mutable_cf_options.compression_opts, cfd->GetID(), cfd->GetName(),
           0 /* level */, false /* is_bottommost */,
           TableFileCreationReason::kRecovery, 0 /* oldest_key_time */,
-          0 /* file_creation_time */, db_id_, db_session_id_,
-          0 /* target_file_size */, meta.fd.GetNumber(), kMaxSequenceNumber);
+          kUnknownNewestKeyTime, 0 /* file_creation_time */, db_id_,
```

Comment:
I think we can use current time as the newest key time here. Otherwise files created during open will not qualify for TTL compaction.

Reply:
updated